binder.md

```diff
@@ -0,0 +1,113 @@
+---
+layout: page_md
+title: The Binder Project
+title_image: assets/logo_binder.svg
+tagline: Reproducible, sharable, interactive computing environments
+permalink: /binder
+---
+
+## What is the Binder Project?
+
+The Binder Project is a community that builds free and open-source tools
```

Comment:
How about something like:

"The Binder project brings the power of customisable and easy to share computing environments to everyone. It allows users to specify custom environments, automates their creation and allows users to share them with a single link. Use-cases involve workshops, scientific workflows and fast sharing of work in teams."

I feel like open-source and free aren't the most important things to clarify here. We are on the Jupyter web site, everything here is open-source and free. I tried to follow the JupyterHub page a bit by trying to explain when/why/who would use Binder. I am imagining that the reader of this first paragraph is someone who is clicking around the website looking for tools that exist, either with a specific use case in mind (do they have something to help me with that?) or just to inform themselves (lets see what they have and what it is good for). This means I think we should focus on use-cases, what pain the user has that we solve and/or what you get when you use binnder.

Reply:
I like it - I've updated with your language. I think it's more content-ful now!